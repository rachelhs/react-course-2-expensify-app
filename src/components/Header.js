import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter, Route, Switch, Link, NavLink } from 'react-router-dom'

const Header = () => (
    <header>
        <h1>Expensify</h1>
        <NavLink to="/" activeClassName="is-active" exact={true}>home</NavLink>
        <NavLink to="/create" activeClassName="is-active">add expense</NavLink>
        <NavLink to="/help" activeClassName="is-active">help</NavLink>
    </header>
);

export default Header;