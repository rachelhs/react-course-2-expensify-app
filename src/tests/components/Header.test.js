import { shallow } from 'enzyme';
import React from 'react';
import Header from '../../components/Header';
import toJSON from 'enzyme-to-json';

test('should render Header correctly', () => {
    const wrapper = shallow(<Header startLogout={()=>{ }} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});
