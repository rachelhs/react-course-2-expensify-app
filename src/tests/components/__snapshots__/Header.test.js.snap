// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <[object Object]
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    home
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    add expense
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/help"
  >
    help
  </[object Object]>
</header>
`;
